servicetalk-concurrent-internal/src/main/java/io/servicetalk/concurrent/internal/ConcurrentUtils.java

```diff
@@ -31,47 +31,44 @@
 
     public static final int CONCURRENT_IDLE = 0;
     public static final int CONCURRENT_EMITTING = 1;
+    private static final int CONCURRENT_PENDING = 2;
```

Comment:
Just wondering why the others need to be public and this can be private ...

Reply:
its a good question. I would like to investigate making them all private and instead expose lock methods, but didn't want to expand the scope of this PR: https://github.com/apple/servicetalk/issues/1023

Reply:
sounds good...